content/ko/donate.html

```diff
@@ -0,0 +1,53 @@
+---
+title: 기부
+linkTitle: "기부하기"
+slug: donate
+top_graphic: 3
+no_donate_footer: true
+lastmod: 2019-01-04
+menu:
+main:
+weight: 30
+parent: donate
```

Comment:
The indentation has been lost between `---` for `menu:`, it should stay the same than https://github.com/letsencrypt/website/blob/master/content/en/donate.html